deploy/kubernetes/delete-driver.sh

```diff
@@ -9,14 +9,10 @@
 set -o nounset
 set -o errexit
 
+readonly DEPLOY_VERSION="${GCE_PD_DRIVER_VERSION:-stable}"
 readonly PKGDIR="${GOPATH}/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver"
+readonly KUSTOMIZE_PATH="${PKGDIR}/bin/kustomize"
 source "${PKGDIR}/deploy/common.sh"
 
-ensure_var GCE_PD_DRIVER_VERSION
-
-readonly KUBEDEPLOY="${PKGDIR}/deploy/kubernetes/${GCE_PD_DRIVER_VERSION}"
-
-kubectl delete -f "${KUBEDEPLOY}/node.yaml" --ignore-not-found
-kubectl delete -f "${KUBEDEPLOY}/controller.yaml" --ignore-not-found
-kubectl delete -f "${KUBEDEPLOY}/setup-cluster.yaml" --ignore-not-found
-kubectl delete secret cloud-sa --ignore-not-found
\ No newline at end of file
+kustomize build ${PKGDIR}/deploy/kubernetes/overlays/${DEPLOY_VERSION} | kubectl delete --ignore-not-found -f -
```

Comment:
how to guarantee kustomize is installed (can we have some func like "ensureKustomize" that installs it if it isn't there).